From: Tommy Webb <tommy@calyxinstitute.org>
Date: Tue, 4 Mar 2025 19:13:05 +0000
Subject: Fix build after "Add bookmark import/export actions"

Make a couple of methods public, since they were internal and not
accessible for use by Cromite.

"Add-menu-item-to-bookmark-all-tabs.patch" already does this, but
we do not include it.

Change-Id: Ia0a18b8dd8472eb9690a124fb715e8b8da40d7e3
---
 .../chrome/browser/native_page/NativePageFactory.java         | 2 +-
 .../browser/download/dialogs/DownloadLocationCustomView.java  | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/chrome/android/java/src/org/chromium/chrome/browser/native_page/NativePageFactory.java b/chrome/android/java/src/org/chromium/chrome/browser/native_page/NativePageFactory.java
--- a/chrome/android/java/src/org/chromium/chrome/browser/native_page/NativePageFactory.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/native_page/NativePageFactory.java
@@ -176,7 +176,7 @@ public class NativePageFactory {
         private final StartupMetricsTracker mStartupMetricsTracker;
 
         public NativePageBuilder(
-                Activity activity,
+                ChromeActivity activity,
                 Supplier<NewTabPageUma> uma,
                 BottomSheetController sheetController,
                 BrowserControlsManager browserControlsManager,
diff --git a/chrome/browser/download/android/java/src/org/chromium/chrome/browser/download/dialogs/DownloadLocationCustomView.java b/chrome/browser/download/android/java/src/org/chromium/chrome/browser/download/dialogs/DownloadLocationCustomView.java
--- a/chrome/browser/download/android/java/src/org/chromium/chrome/browser/download/dialogs/DownloadLocationCustomView.java
+++ b/chrome/browser/download/android/java/src/org/chromium/chrome/browser/download/dialogs/DownloadLocationCustomView.java
@@ -85,7 +85,7 @@ public class DownloadLocationCustomView extends ScrollView
         mDirectoryAdapter.update();
     }
 
-    void setTitle(CharSequence title) {
+    public void setTitle(CharSequence title) {
         mTitle.setText(title);
     }
 
@@ -93,7 +93,7 @@ public class DownloadLocationCustomView extends ScrollView
         mSubtitleView.setText(subtitle);
     }
 
-    void setFileName(CharSequence fileName) {
+    public void setFileName(CharSequence fileName) {
         mFileName.setText(fileName);
     }
 
-- 

